### DISCUSSION TITLE
Struggling to modify parameters through the Python interface

### DISCUSSION THREAD

**Discussion by BDELETREE** [InteractiveComputerGraphics/SPlisHSPlasH#162] [Q&A]
Thank you for this great and impressive work !
This is probably something trivial, but I am struggling at changing simulation parameters through the python interface.
For instance I am intending to change the resolution method before launching the Gui, I made various trials without success.
Could you give me an example showing the appropriate  way to do so?

**Comment by janbender** [comment]
What exactly do you mean by resolution method?

**Reply by BDELETREE** [reply]
Thank you janbender,

I meant change the simulation method actually.

I figured out my mistake : 
I was using  
```python
sim = sph.Simulation.getCurrent()
sim.setValueInt(sim.SIMULATION_METHOD, method_Int)
```

Without having initiated the simulation before with:
```python 
base.initSimulation()
```
Everything works fine now.
